update_docs.py

```diff
@@ -0,0 +1,229 @@
+    'uno' : { 'markdown': 'uno.md', 'code' : 'pettingzoo/classic/rlcard_envs/uno.py', 'environment' : (uno_v4.env())},
+    'adversarial_pursuit' : { 'markdown': 'adversarial_pursuit.md', 'code' : 'pettingzoo/magent/adversarial_pursuit_v3.py', 'environment' : (adversarial_pursuit_v3.make_env())},
+    'battle' : { 'markdown': 'battle.md', 'code' : 'pettingzoo/magent/battle_v3.py', 'environment' : (battle_v3.make_env())},
+    'battlefield' : { 'markdown': 'battlefield.md', 'code' : 'pettingzoo/magent/battlefield_v3.py', 'environment' : (battlefield_v3.make_env())},
+    'combined_arms' : { 'markdown': 'combined_arms.md', 'code' : 'pettingzoo/magent/combined_arms_v5.py', 'environment' : (combined_arms_v5.make_env())},
+    'gather' : { 'markdown': 'gather.md', 'code' : 'pettingzoo/magent/gather_v3.py', 'environment' : (gather_v3.make_env())},
+    'tiger_deer' : { 'markdown': 'tiger_deer.md', 'code' : 'pettingzoo/magent/tiger_deer_v3.py', 'environment' : (tiger_deer_v3.make_env())},
+    'simple_adversary' : { 'markdown': 'simple_adversary.md', 'code' : 'pettingzoo/mpe/simple_adversary_v2.py', 'environment' : (simple_adversary_v2.make_env())},
+    'simple_crypto' : { 'markdown': 'simple_crypto.md', 'code' : 'pettingzoo/mpe/simple_crypto.py', 'environment' : (simple_crypto_v2.make_env())},
+    'simple_push' : { 'markdown': 'simple_push.md', 'code' : 'pettingzoo/mpe/simple_push_v2.py', 'environment' : (simple_push_v2.make_env())},
+    'simple_reference' : { 'markdown': 'simple_reference.md', 'code' : 'pettingzoo/mpe/simple_reference_v2.py', 'environment' : (simple_reference_v2.make_env())},
+    'simple_speaker_listener' : { 'markdown': 'simple_speaker_listener.md', 'code' : 'pettingzoo/mpe/simple_speaker_listener_v3.py', 'environment' : (simple_speaker_listener_v3.make_env())},
+    'simple_spread' : { 'markdown': 'simple_spread.md', 'code' : 'pettingzoo/mpe/simple_spread_v2.py', 'environment' : (simple_spread_v2.make_env())},
+    'simple_tag' : { 'markdown': 'simple_tag.md', 'code' : 'pettingzoo/mpe/simple_tag_v2.py', 'environment' : (simple_tag_v2.make_env())},
+    'simple' : { 'markdown': 'simple.md', 'code' : 'pettingzoo/mpe/simple_v2.py', 'environment' : (simple_v2.make_env())},
+    'simple_world_comm' : { 'markdown': 'simple_world_comm.md', 'code' : 'pettingzoo/mpe/simple_world_comm_v2.py', 'environment' : (simple_world_comm_v2.make_env())},
+    'multiwalker' : { 'markdown': 'multiwalker.md', 'code' : 'pettingzoo/sisl/multiwalker/multiwalker.py', 'environment' : (multiwalker_v7.env())},
+    'pursuit' : { 'markdown': 'pursuit.md', 'code' : 'pettingzoo/sisl/pursuit/pursuit.py', 'environment' : (pursuit_v4.env())},
+    'waterworld' : { 'markdown': 'waterworld.md', 'code' : 'pettingzoo/sisl/waterworld/waterworld.py', 'environment' : (waterworld_v3.env())}
+}
+for file in files:
+
+    file_content = [ line for line in open(files[file]['markdown']) ]
```

Comment:
Why are we reading the markdown file? Shouldn't we be trying to write the markdown file with only information provided by the environment values and comments?